### DISCUSSION TITLE
Bug with extensible enum's

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#715] [Q&A] [answered]
When product page loaded, only the first 200 of extensible attribute value labels are loaded other half display its id instead
![image](https://github.com/atrocore/atrocore/assets/30953478/d49c80ee-35a2-450e-b335-27fa91d68573)

**Comment by rratsun** [comment]
We will check and fix soon.

**Comment by zssvaidar** [comment]
![Screenshot from 2023-05-15 15-47-59](https://github.com/atrocore/atrocore/assets/30953478/04f21f87-1952-4c6a-8c1f-48994763065e)
![Screenshot from 2023-05-15 15-47-51](https://github.com/atrocore/atrocore/assets/30953478/b161a6d2-73eb-414e-964a-2a38df4b4822)
![Screenshot from 2023-05-15 15-47-44](https://github.com/atrocore/atrocore/assets/30953478/1e5185f3-efee-4400-a046-c68584ab495d)
Little context on local machine everything is fine bit on hosted machine enum's limited to 200

**Reply by rratsun** [reply]
We will increase the number of options to 5000, but in your case that does not help, because I see you have the List with more than 56000 options. In your case, you need to use Relation, not List. Unfortunately, so far, we don't have attribute type Link and MultipleLink for it, but we can develop such attribute type. The price of development is 400 EUR. 

For now, I recommend to you to create new Entity and create a relationship between Product and your new Entity and use this field for your needs.

**Comment by rratsun** [comment, marked as answer]
We have increased the max number of options to 5000 in Core 1.5.64
